libcontainer/selinux/selinux.go

```diff
@@ -397,22 +401,30 @@ func GetLxcContexts() (processLabel string, fileLabel string) {
 				processLabel = strings.Trim(val, "\"")
 			}
 			if key == "file" {
-				fileLabel = strings.Trim(val, "\"")
+				scon := NewContext(strings.Trim(val, "\""))
+				scon["level"] = "s0:c1023" // No container can read this level
+				fileLabel = scon.Get()
 			}
 		}
 	}
+exit:
+	return processLabel, fileLabel
+}
 
-	if processLabel == "" || fileLabel == "" {
+func GetLxcContexts() (processLabel string, fileLabel string) {
+	if !selinuxEnabledHost {
 		return "", ""
 	}
 
-exit:
-	//	mcs := IntToMcs(os.Getpid(), 1024)
-	mcs := uniqMcs(1024)
-	scon := NewContext(processLabel)
+	if !selinuxEnabled {
+		return "", defFileLabel
+	}
+
+	mcs := uniqMcs(1023)
```

Comment:
The code that we replaced has 1024 whereas we changed it to 1023 here. Can we add a comment?